repo/migrations/Migration011.go

```diff
@@ -0,0 +1,190 @@
+package migrations
+
+import (
+	"database/sql"
+	"fmt"
+	"github.com/OpenBazaar/wallet-interface"
+	"path"
+	"strings"
+)
+
+var WalletCoinType wallet.CoinType
+
+type Migration011 struct{}
+
+func (Migration011) Up(repoPath, databasePassword string, testnetEnabled bool) error {
+	var (
+		databaseFilePath string
+	)
+	if testnetEnabled {
+		databaseFilePath = path.Join(repoPath, "datastore", "testnet.db")
+	} else {
+		databaseFilePath = path.Join(repoPath, "datastore", "mainnet.db")
+	}
+
+	const (
+		alterKeysSQL                 = "alter table keys add coin text;"
```

Comment:
Could we use `coinType` to match the column names we're using in `purchases` and `sales`?